build/syntax.go

```diff
@@ -273,6 +273,51 @@ func (x *StringExpr) Copy() Expr {
 	return &n
 }
 
+type MultiPartStringExpr struct {
```

Comment:
This isn't the most useful struct. We probably want some niceties like getting the decoded value our of it. Might be nice to iterate as we build out the SDK.

Reply:
Yeah or we just kill implicit string concatenation, it's not particularly nice anyway - I wasn't gonna have it initially but was having a big pain migrating.